src/odemis/odemisd/main.py

```diff
@@ -87,6 +90,40 @@ def __init__(self, model_file, create_sub_containers=False,
             logging.exception("When instantiating file %s", self._model.name)
             raise IOError("Unexpected error at instantiation")
 
+    def _on_va_change(self, _):
+        """
+        Callback to VA change, calls write_persistent_data
+        """
+        self._write_persistent_data()
```

Comment:
Didn't we want to write it to a file in case odemis crashes? Otherwise, what's the point of saving it in a dictionary, if we only write the dictionary to a file in the end?